testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/ServiceTest.scala

```diff
@@ -293,19 +373,24 @@ object ServiceTest {
     new TestServer(assignedPort, application, srv)
   }
 
-  private def disableModules(builder: GuiceApplicationBuilder, classes: String*): GuiceApplicationBuilder = {
-    val loadedClasses = classes.flatMap { className =>
-      try {
-        Seq(getClass.getClassLoader.loadClass(className))
-      } catch {
-        case cfne: ClassNotFoundException =>
-          Seq.empty[Class[_]]
+  /**
+   * Enriches [[GuiceApplicationBuilder]] with a `disableModules` method.
+   */
+  private implicit class GuiceBuilderOps(val builder: GuiceApplicationBuilder) extends AnyVal {
```

Comment:
Method extensions is a good option for this kind of API. We just add one more method to the already existing fluent API.